.circleci/config.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       - run:
           name: Run the integration tests
           command: ./scripts/run-integration-tests.sh
-          no_output_timeout: 15m
+          no_output_timeout: 20m
```

Comment:
Oh, it's that slow?

Reply:
eksctl can sometimes take longer than 15m to create the cluster and mng, so it did timeout once. Haven't seen any problems after moving it to 20m!